src/NServiceBus.Testing/Handler.cs

```diff
@@ -105,16 +106,16 @@ public Handler<T> ExpectNotSendLocal<TMessage>(Func<TMessage, bool> check)
         /// </summary>
         public Handler<T> ExpectReturn<TEnum>(Func<TEnum, bool> check)
```

Comment:
This is the only remaining case without a test. I can't find previous usage. What does this test for?